service/src/main/java/com/excilys/service/importTool/impl/ComputerImportService.java

```
@@ -0,0 +1,152 @@
+package com.excilys.service.importTool.impl;
+
+import au.com.bytecode.opencsv.CSVReader;
+import com.excilys.binding.validation.ComputerValidator;
+import com.excilys.core.conflict.Rapport;
+import com.excilys.core.conflict.format.Error;
+import com.excilys.core.conflict.format.ErrorMessage;
+import com.excilys.core.conflict.format.Fields;
+import com.excilys.core.dto.ComputerDTO;
+import com.excilys.service.importTool.IComputerImportService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class ComputerImportService implements IComputerImportService {
+
+    @Autowired
+    private ComputerValidator computerValidator;
+
+    @Override
+    public Rapport importComputersFromCSV(MultipartFile file) {
+        Rapport rapport = new Rapport();
+        File convFile = new File(file.getOriginalFilename());
+        try {
+            file.transferTo(convFile);
+        } catch (IOException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
```

Throw a runtimeException or ignore the exception